pipeline/vars/common.groovy

```diff
@@ -219,8 +218,20 @@ def sendEMail(def subjectPrefix, def test_results, def isStage=true) {
     /*
         Send an email notification.
     */
+    def versionFileExists = sh(
+        returnStatus: true, script: "ls -l /ceph/cephci-jenkins/latest-rhceph-container-info/version_info.json"
+    )
+    if (versionFileExists == 0) {
+        version_info = jsonToMap("/ceph/cephci-jenkins/latest-rhceph-container-info/version_info.json")
```

Comment:
I prefer not to refer this location due to race conditions as the file is not prefixed or suffixed with the version.